### PR TITLE
Add links to validmind.ai

### PR DESCRIPTION
**Pull request by nrichers**
## Internal Notes for Reviewers

This PR adds links to validmind.ai to top navigation and footer for our docs site:

**Top navigation:**

 image

**Footer:**

image

### CODE REVIEW

File: site/_site/index.html [hunk 1]
```diff
@@ -534,6 +536,9 @@ <h2 style="color: var(--bs-white);" class="anchored">Support &amp; Training</h2>
     <div class="nav-footer-right">
       <ul class="footer-items list-unstyled">
     <li class="nav-item">
+    <a class="nav-link" href="https://validmind.com/">Main Site <i class="fa-solid fa-external-link" aria-label="external-link"></i></a>
```
Comment (gtagle) [inline site/_site/index.html:539]: Not too sure, but should this also be "validmind.ai" ?

File: site/_site/releases/2023-sep-27/enhancement.html [hunk 2]
```diff
@@ -159,8 +159,28 @@
     <a class="nav-link" href="../../guide/support.html" rel="" target="">
  <span class="menu-text">Support</span></a>
   </li>  
+  <li class="nav-item">
+    <a class="nav-link" href="https://validmind.com/" rel="" target="_blank">
+ <span class="menu-text">Main Site <i class="fa-solid fa-external-link" aria-label="external-link"></i></span></a>
```
Comment (gtagle) [inline site/_site/releases/2023-sep-27/enhancement.html:164]: Should this also be "validmind.ai" ?

File: site/_site/releases/2023-sep-27/enhancement.html [hunk 3]
```diff
@@ -159,8 +159,28 @@
...
+  </li>  
 </ul>
             <div class="quarto-navbar-tools ms-auto">
+    <div class="dropdown">
+      <a href="" title="" id="quarto-navigation-tool-dropdown-0" class="quarto-navigation-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false" aria-label=""><i class="bi bi-globe"></i></a>
+      <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="quarto-navigation-tool-dropdown-0">
+          <li>
+            <a class="dropdown-item quarto-navbar-tools ms-auto-item" href="https://validmind.com/">
+            Main Site {{&lt; fa external-link &gt;}}
```
Comment (gtagle) [inline site/_site/releases/2023-sep-27/enhancement.html:173]: Should this also be "validmind.ai" ?

File: site/_site/releases/2023-sep-27/enhancement.html [hunk 4]
```diff
@@ -689,6 +707,9 @@ <h1 class="title">Enhancements – September 26, 2023</h1>
     <div class="nav-footer-right">
       <ul class="footer-items list-unstyled">
     <li class="nav-item">
+    <a class="nav-link" href="https://validmind.com/">Main Site <i class="fa-solid fa-external-link" aria-label="external-link"></i></a>
```
Comment (gtagle) [inline site/_site/releases/2023-sep-27/enhancement.html:710]: Should this also be "validmind.ai" ?

**Review by gtagle — CHANGES_REQUESTED** [review]
I found a few mentions of "Main Site", commented on some of them but not all.

**Comment by nrichers** [comment]
Ah, I forgot to re-render the docs site after my last change, fixed. There should be no references to "Main Site" in our docs site now:

**Review by gtagle — APPROVED** [review]
Good to go!

**Pull request merged**